### DISCUSSION TITLE
链接无法添加文本

### DISCUSSION THREAD

**Discussion by JCVICTORSUN** [LIlGG/halo-theme-sakura#384] [Q&A]
### 目前使用的 Sakura 版本？

2.0.0-beta.2

### 目前使用的 halo 版本？

2.7.0

### 建议/问题

![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/b6d3f026-8756-4933-a44a-6480aad1e88d)
如图，这是preview显示的界面，在链接上方有自己加的文本，但是发布之后却没有
![image](https://github.com/LIlGG/halo-theme-sakura/assets/53136539/5517aceb-e6e9-4726-a81b-31dcb0c08aa6)
自定义模板选择默认或者友情链接都不显示）
编辑页面的时候也是有字的，另外还发现了一个问题就是如果使用别家的编辑器（Wangediter），即使添加了文字，发布之后再点击编辑页面文字都不见了。不确定是不是同一个问题

**Comment by LIlGG** [comment]
如果选择的是自定义模板，那么就不能使用 `links` 路由。需要使用你所创建的自定义页面的路由地址。如 `https://halo2.lixingyong.com/friend` 所示。
